components/RepaymentInfo/RepaymentInfo.tsx

```diff
@@ -14,6 +17,27 @@ export function RepaymentInfo({ loan }: RepaymentInfoProps) {
     longFormattedInterestRate,
     longFormattedEstimatedPaybackAtMaturity,
   } = useLoanDetails(loan);
+
+  const maturityDate = useMemo(
+    () =>
+      loan.endDateTimestamp
+        ? dayjs.unix(loan.endDateTimestamp).format('MMMM D, YYYY')
+        : '',
+    [loan],
+  );
+
+  const createCalEvent = useCallback(() => {
+    const config: CalendarOptions = {
+      title: `Backed: Loan #${loan.id} ${loan.collateralName} due`,
+      description: `Loan repayment of ${longFormattedEstimatedPaybackAtMaturity} is due: https://www.withbacked.xyz/loans/${loan.id}. Note that this due date could change if the loan is bought out.`,
+      start: new Date(loan.endDateTimestamp! * 1000),
+    };
+
+    const googleCalendar = new GoogleCalendar(config);
+    const link = googleCalendar.render();
```

Comment:
I think you could just create the address and have a regular link in the page rather than doing `window.open` (which could get blocked by the browser as a popup)